src/plugins/midi-piano/midi-piano.yml

```diff
@@ -0,0 +1,16 @@
+name:
+  en: Midi piano
+  de: Midi-Klavier
+midiTrackTitle:
+  en: Track title
+  de: Stück-Titel
```

Comment:
For cases where we deal with tracks, the translation we use in German is "Spur". Would this match here too?

Reply:
It's the title of the whole piece like "Edvard Grieg: Piano Concerto op. 16, I." Maybe better "Title of piece"? What would you suggest?

Reply:
Ok, I see. So then since there aren't multiple tracks/pieces, I'd go simply just with "Title", like in the other plugins.